src/components/AdminDashboard/AntTableComponents/AntTable.js

```diff
@@ -1,50 +1,52 @@
-import React, { useState, useEffect } from 'react';
-import { Table, Space, Divider } from 'antd';
-import axios from 'axios';
-import { getData } from '../../../utils/DashboardHelperFunctions';
+import React, { useEffect, useState } from 'react';
+import { Table } from 'antd';
 import CompleteIncident from '../CompleteIncident';
 import { DateTime } from 'luxon';
 
 //https://ant.design/components/table/ <---documentation on the table
 
+/**
+ * @typedef AntTableProps
+ * @property {any[]} incidents - incident data
+ * @property {number[]} selectedIds - an aray of the currently selected incident IDs
+ * @property {React.Dispatch<React.SetStateAction<number[]>} setSelectedIds - setter for selected IDs
+ * @property {boolean} showConfidence - shows a column for incident confidence
+ */
+
+/**
+ * Component for rendering a list of incident data on the AdminDashboard
+ *
+ * @param {AntTableProps} props
+ * @returns {JSX.Element} the AntTable component
+ */
```

Comment:
Where did you get the format for these doc comments? It's pretty cool.

Reply:
I believe it's JSDoc.

Reply:
https://jsdoc.app/

The best part of it is that it enables better intellisense/completion because the IDE can tell what types your objects are.

![code](https://res.cloudinary.com/hvvmlo7os/image/upload/v1631905825/jsdocint_fho930.png)

Reply:
Oh dang. That's sweet.